tests/ttnn/conftest.py

```diff
@@ -48,3 +48,21 @@ def pre_and_post(request):
 
     ttnn.tracer.disable_tracing()
     ttnn.CONFIG = original_config
+
+
+def pytest_addoption(parser):
```

Comment:
Did you want to move this up to top-level conftest.py?

Reply:
ah yes, I didn't notice there is another conftest.py. I'll move it to there